features/netsocket/EMACInterface.h

```diff
@@ -154,12 +154,13 @@ class EMACInterface : public virtual NetworkInterface
         return this;
     }
 
-protected:
```

Comment:
Do you really need this method exposed? You are deriving from `EMACInterface`, so you can get at this anyway. Was this something to do with the bridge?

Also, this is a virtual override of a protected function anyway in the base class, so exposing it more in a derived class is a bit weird.